src/StaticCategory/StaticCategory.js

```diff
@@ -1,16 +1,25 @@
 import { categoryIcons } from './categoryIcons.js'
 import StaticItem from '../StaticItem/StaticItem'
+import { BsPlusCircle } from 'react-icons/bs'
```

Comment:
Love the change to the plus! @jaypeasee - hoping we can add a hover state that tells the user what it does!